### PR TITLE
Test CircleCI Win

### PR DESCRIPTION
**Pull request by Vulfox**
Creation of CircleCI Testing. This may not end well.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -8,4 +8,4 @@ jobs:
     steps:
       # Commands are run in a Windows virtual machine environment
         - checkout
-        - run: Write-Host 'Hello, Windows'
+        - run: Write-Host 'Hello, Windows!'
```
Comment (robloxrob) [inline .circleci/config.yml:11]: `!`

File: .github/workflows/test-driver.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [ master ]
```
Comment (shishir-a412ed) [inline .github/workflows/test-driver.yml:5]: Suggestion: You can also add `workflow_dispatch` which will allow users to manually trigger GHA from the actions tab.

```
on:
  pull_request:
    branches: [ master ]

  # Allows you to run this workflow manually from the Actions tab
  workflow_dispatch:
```

File: .github/workflows/test-driver.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: windows-2016
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Prerequisites (PFX, IIS)
+        run: |
```
Comment (shishir-a412ed) [inline .github/workflows/test-driver.yml:14]: Don't think you need `|` for a single command.

File: .github/workflows/test-driver.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+          Import-PfxCertificate -FilePath .\test\test.pfx -CertStoreLocation Cert:\\LocalMachine\\My -Password (ConvertTo-SecureString -String 'Test123!' -AsPlainText -Force)
+        shell: powershell
+      - name: Run iis-driver integration tests
+        run: |
```
Comment (shishir-a412ed) [inline .github/workflows/test-driver.yml:18]: Same as above. Can you try it without `|`?

**Review by shishir-a412ed — APPROVED** [review]
LGTM!

**Pull request merged**